### ISSUE TITLE
npm errors

### ISSUE THREAD

**Issue by jfhbrook** [tautologistics/node-htmlparser#3]
Hey there,

I tried installing htmlparser with npm, and it had some issues:

http://gist.github.com/476153

Thanks!

**Comment by tautologistics** [comment]
Strange. I just uninstalled and installed again and it works just fine. It looks like this was a temporary issue with the npm repository.

Would you mind giving it a try again? Also, have you pasted the same error log to the npm google group?

**Comment by jfhbrook** [comment]
Hmm! Maybe it has to do with the fact that I'm installing with su -c'' instead of as a regular user.

I just tried again, and it didn't work this time either.

I haven't reported the issue to npm, but this was because a friend had what I thought was a similar problem with [his node package](http://github.com/substack/dnode) and it had something to do with validation on his end. I didn't really ask him for details. It was my impression, though, that it wasn't an npm error. That, and I just bugged isaacs about an npm error with coffee-script. I'd feel bad giving him another. ;)
